lib/primitive/lib/Cardano/Wallet/Primitive/Types/Certificates.hs

```diff
@@ -165,24 +183,39 @@ instance Buildable PoolRetirementCertificate where
 data NonWalletCertificate
     = GenesisCertificate
     | MIRCertificate
+    | CommitteeHotKeyAuthorization
+    | CommitteeColdResignation
+    | DRepRegistration
+    | DRepDeregistration
```

Comment:
```suggestion
    | AuthCommitteeHotKey
    | ResignCommitteeColdKey
    | RegDRep
    | UnRegDRep
```

Again, I want our naming conventions to match exactly. In this case, they match the naming convention for the patterns in `Cardano.Ledger.Api.Tx.Cert`.

Reply:
done